submitParticipantData.json

```diff
@@ -27,6 +27,10 @@
         "values": [628733063, 319570637, 178420302],
         "dataType": "number"
     },
+    "state.253532712": {
+        "values": [723775357, 153444133, 572474909, 308427446, 211228524, 277568192, 611398522, 181769837],
```

Comment:
should 178420302 be on the list too?

Reply:
Thanks for the catch, going to add it now